echeck.c

```diff
@@ -3804,7 +3817,7 @@ void check_money(bool do_move)
           if (t->ship == i) {
             if (t->hasmoved > 1) {      /* schon bewegt! */
               sprintf(warn_buf, errtxt[UNITONSHIPHASMOVED], uid(t), itob(i));
-              Error(warn_buf, t->line_no, t->long_order);
```

Comment:
Das hier schient mir der eigentliche Bugfix zu sein, alles andere in diesem commit dient nur der Verwirrung des Reviewers (und macht mir einen Haufen Konflikte beim Cherry-Picking).